src/modules/prpn/LSTMCell.py

```diff
@@ -0,0 +1,47 @@
+import torch.nn.functional as F
+import torch.nn as nn
+from torch.nn.modules.rnn import *
+
```

Comment:
Add a comment explaining what this file is for. If any of it is reused from PRPN, say so/give credit. If any of it already exists in AllenNLP or PyTorch, either switch to that version, or at least make it clear that you need to use this version to match the PRPN paper in some way.

Reply:
Same with the rest of this directory.